yarn-project/noir-contracts/src/scripts/copy_output.ts

```diff
@@ -20,6 +20,8 @@ const PROJECT_CONTRACTS = [
   { name: 'EcdsaAccount', target: '../aztec.js/src/abis/', exclude: [] },
 ];
 
+const INTERFACE_CONTRACTS = ['test'];
```

Comment:
So we specify here which contracts we'd like to generate an interface for?

How would a user do it? I guess this is a question to be answered in https://github.com/AztecProtocol/aztec-packages/issues/1457 :)

Reply:
Tagging @sirasistant who's working on that issue, for awareness of this.

Reply:
Discussed with @sirasistant and seems I'll be taking on that one. This script for now is tailored specifically for our use, so I think it's fine. I'll make sure to strip this out when promoting it.